onnx/common/ir_pb_converter.cc

```diff
@@ -483,6 +486,17 @@ void encodeGraph(ONNX_NAMESPACE::GraphProto * p_g, const std::shared_ptr<Graph>
     }
     for(auto output : node->outputs()) {
       p_n->add_output(value_name(output));
+
+      // only save it if it has content
```

Comment:
To be more clear here, we should say, the output information (including shape/type) is already stored in graph's output.

Reply:
++